jose/jwt.py

```diff
@@ -418,12 +418,11 @@ def _validate_at_hash(claims, access_token, algorithm):
     """
```

Comment:
Nit: Can you update the docstring for this method to pull the explanation from the RFC, similar to the other validation methods?

```
    """
    Validates that the 'at_hash' is valid.
    
    Its value is the base64url encoding of the left-most half of the hash
    of the octets of the ASCII representation of the access_token value,
    where the hash algorithm used is the hash algorithm used in the alg
    Header Parameter of the ID Token's JOSE Header. For instance, if the
    alg is RS256, hash the access_token value with SHA-256, then take the
    left-most 128 bits and base64url encode them. The at_hash value is a
    case sensitive string.  Use of this claim is OPTIONAL.

    Args:
        claims (dict): The claims dictionary to validate.
        access_token (str): The access token returned by the OpenID Provider.
        algorithm (str): The algorithm used to sign the JWT, as specified by
            the token headers.
    """
```